skactiveml/pool/multi/_wrapper.py

```diff
@@ -0,0 +1,335 @@
+                 random_state=None):
+        super().__init__(random_state=random_state)
+        self.strategy = strategy
+        self.n_annotators = n_annotators
+        self.y_aggregate = y_aggregate
+
+    def query(self, X_cand, query_params_dict=None, A_cand=None, batch_size=1,
+              pref_annotators_per_sample=1, A_pref=None,
+              return_utilities=False):
+
+        """Determines which candidate sample is to be annotated by which
+        annotator. The samples are first and primarily ranked by the given
+        strategy as if one unspecified annotator where to annotate the sample.
+        Then for each sample the sample-annotator pairs are ranked based either
+        on previously set preferences or at random.
+
+        Parameters
+        ----------
+        X_cand : array-like, shape (n_samples, n_features)
+            Candidate samples from which the strategy can select.
+        A_cand : array-like, shape (n_samples, n_annotators), optional
+        (default=None)
+            Boolean matrix where `A_cand[i,j] = True` indicates that
+            annotator `j` can be selected for annotating sample `X_cand[i]`,
+            while `A_cand[i,j] = False` indicates that annotator `j` cannot be
+            selected for annotating sample `X_cand[i]`. If `A_cand is None`, each
+            annotator is assumed to be available for labeling each sample.
+            `A_cand` must only be None, if `n_annotators is non None`.
+        query_params_dict : dict, default=None
+            Dictionary for the parameters of the query method.
+            If `dict` contains a value for `y`, it is transformed by
+            `y_aggregate`. If `query_params_dict is None` an empty
+            dictionary is used.
+        batch_size : 'adaptive'|int, optional (default=1)
+            The number of samples to be selected in one AL cycle. If 'adaptive'
+            is set, the `batch_size` is set to 1.
+        A_pref : array-like, shape (n_samples, n_annotators) or
+                  (n_annotators,) optional (default=None)
+            The preferred ranking of each annotator.
+            1.) If `A_pref` is of shape (n_samples, n_annotators) for each
+             sample `i` the value-annotators pair `(i, j)` is preferably picked
+             over the pair `(i, k)` if `A_pref[i, j]` is greater or
+             equal to `A_pref[i, k]`.
+            2.) If `A_pref` is of shape (n_annotators,) for each sample
+            `i` the value-annotators pair `(i, j)` is preferentially picked
+             over the pair `(i, k)` if `A_pref[j]` is greater or
+             equal to `A_pref[k]`.
+            3.) If `A_pref` is None, the annotators are chosen at random, with
+             a different distribution for each sample.
+        return_utilities : bool, optional (default=False)
+            If true, also returns the utilities based on the query strategy.
+        pref_annotators_per_sample : int, array-like, optional (default=1)
```

Comment:
What is a proper name of `pref_annotators_per_sample`:
- reasons for naming: the number of selected can vary because of `A_cand` and `batch_size`, there are case where more or less annotators per sample are queried
- problem: preferred is fuzzy -> the user cannot really understand its meaning without documentation
- solution: `n_annotators_per_sample` ? Other proposals are welcome ;)